servermaster/jobmanager.go

```diff
@@ -108,12 +111,20 @@ func (jm *JobManagerImplV2) SubmitJob(ctx context.Context, req *pb.SubmitJobRequ
 		id  lib.WorkerID
 		err error
 	)
+
+	// [TODO] check the ctx
+	epoch, err := jm.epochGen.GenerateEpoch(ctx)
+	if err != nil {
+		resp.Err = derrors.ToPBError(err)
+		return resp
+	}
+
 	meta := &lib.MasterMetaKVData{
 		// TODO: we can use job name provided from user, but we must check the
 		// job name is unique before using it.
-		ID:         jm.uuidGen.NewString(),
-		Config:     req.Config,
-		StatusCode: lib.MasterStatusUninit,
+		ID:     jm.uuidGen.NewString(),
+		Epoch:  epoch,
+		Config: req.Config,
 	}
```

Comment:
Is `StatusCode: lib.MasterStatusUninit` missing?

Reply:
Thanks. Bad code coverage.